nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/resources/OSGI-INF/TagService.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0"?>
 <component name="org.nuxeo.ecm.platform.tag.TagService">
 
+  <require>org.nuxeo.tag.service.properties</require>
```

Comment:
I don't like this either. If a customer overrides the property we won't be able to depend on its component whose id we don't know. So the overall ordering must be done differently.